.github/workflows/fast_testing.yml

```diff
@@ -10,6 +10,40 @@ on:
       - '*'
 
 jobs:
+  version_check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone the module
+        uses: actions/checkout@v2
+
+      - name: Setup tarantool
+        uses: tarantool/setup-tarantool@v1
```

Comment:
"Node.js 12 actions are deprecated. Please update the following actions to use Node.js 16: actions/checkout@v2, tarantool/setup-tarantool@v1."

Reply:
checkout v3 and setup-tarantool v2 are based on Node 16